### DISCUSSION TITLE
Announce speed limit when warning of radar camera

### DISCUSSION THREAD

**Discussion by TobiPeterG** [osmandapp/OsmAnd#20927] [Ideas]
Currently, OSMand warns of radar cameras (great), but doesn't announce the allowed speed limit (not so great). It would be nice if this could be activated :)

**Comment by DmitryAlexei** [comment]
You can configure speed limit tolerance in the voice prompts menu
for more information see https://osmand.net/docs/user/navigation/guidance/voice-navigation#speed-limit
